packages/core/src/shared/utilities/timeoutUtils.ts

```diff
@@ -219,40 +219,102 @@ interface WaitUntilOptions {
     readonly interval?: number
     /** Wait for "truthy" result, else wait for any defined result including `false` (default: true) */
     readonly truthy?: boolean
+    /** A backoff multiplier for how long the next interval will be (default: None, i.e 1) */
+    readonly backoff?: number
+    /**
+     * Only retries when an error is thrown, otherwise returning the result regardless of truthiness.
+     * - Ignores 'truthy' arg
+     * - If the timeout is reached it will throw the last error
+     * - default: false
+     */
+    readonly retryOnFail?: boolean
 }
 
+export const waitUntilDefaultTimeout = 2000
+export const waitUntilDefaultInterval = 500
+
 /**
  * Invokes `fn()` until it returns a truthy value (or non-undefined if `truthy:false`).
  *
+ * Also look at {@link withRetries} to be able to retry on failures.
```

Comment:
I guess this comments no longer relevant since it looks like withRetries is now removed

Reply:
Thanks for catching, fixed